sound/soc/sof/ipc4-topology.c

```diff
@@ -1635,16 +1639,21 @@ static int sof_ipc4_prepare_process_module(struct snd_sof_widget *swidget,
 	struct sof_ipc4_process *process = swidget->private;
 	struct sof_ipc4_available_audio_format *available_fmt = &process->available_fmt;
 	struct sof_ipc4_control_data *control_data;
+	struct sof_ipc4_fw_module *fw_module;
 	void *cfg = process->ipc_config_data;
 	int ret;
 
 	available_fmt->ref_audio_fmt = &available_fmt->base_config->audio_fmt;
 
 	/*
-	 * Output format is optional for process modules.
-	 * Process modules setup the output format based on audio format tokens in topology.
+	 * If a module is from external library, it is a loadable module,
+	 * otherwise, it is a basefw module. Output format is optional
```

Comment:
I'm sorry this logic is inverted. On the contrary, if a module is not in the base FW, it is automatially loaded from an external library